src/ui/account-view.cpp

```diff
@@ -97,7 +97,7 @@ void AccountView::updateAccountInfoDisplay()
                                "\"color:#A4A4A4; text-decoration: none;\" "
                                "href=\"%1\">%2</a>").arg(href).arg(host);
 
-        mServerAddr->setText(text);
+        mServerAddr->setText(account.isPro() ? tr("%1 <small>pro version</small>").arg(text) : text);
```

Comment:
Maybe `QString("%1 <small>%2</small").arg(text).arg(tr("pro version"))` is more suitable for translation?